internal/stacks/stackruntime/internal/stackeval/component_config.go

```diff
@@ -410,183 +407,3 @@ func (c *ComponentConfig) reportNamedPromises(cb func(id promising.PromiseID, na
 	cb(c.validate.PromiseID(), c.Addr().String())
 	cb(c.moduleTree.PromiseID(), c.Addr().String()+" modules")
 }
-
-// sourceBundleModuleWalker is an implementation of [configs.ModuleWalker]
-// that loads all modules from a single source bundle.
-type sourceBundleModuleWalker struct {
```

Comment:
Just to double check, these were just moved, right?

Reply:
That's right.